mitodl/ocw-hugo-themes
# render resource collections

#### Pre-Flight checklist


#### What are the relevant tickets?

closes #279

#### What's this PR do?

This adds support for rendering resource collections in ocw-www. This entails:

1. a hugo template which uses the recently-added (#400) `content_map.json` file to build a map from resource UUID to resource metadata, alongside fetching some other data which is needed (like course `data.json` files).
2. React code to pull out the values from `window` and then render out a list of resources using components we already have hanging around.

#### How should this be manually tested?

If you run `start:www` with a locally checked out copy of the RC `ocw-www` site you should be able to check out http://localhost:3000/resource_collections/open-textbooks/ for an example of a resource collection. The links to the resource's page should work, as should links to the search from topics.

#### screen

<img width="1557" alt="Screen Shot 2022-02-09 at 3 58 36 PM" src="https://user-images.githubusercontent.com/6207644/153288945-504c8e03-f0b8-4ac5-b5a9-3c295bbe0c94.png">
